chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/data/HistoricalStatisticsCreator.java

```diff
@@ -14,33 +17,46 @@
 import org.springframework.stereotype.Component;
 import org.springframework.transaction.annotation.Transactional;
 
+import gov.healthit.chpl.dao.CertificationStatusEventDAO;
 import gov.healthit.chpl.dao.CertifiedProductDAO;
 import gov.healthit.chpl.dao.statistics.DeveloperStatisticsDAO;
 import gov.healthit.chpl.dao.statistics.ListingStatisticsDAO;
 import gov.healthit.chpl.dao.statistics.SurveillanceStatisticsDAO;
 import gov.healthit.chpl.domain.DateRange;
+import gov.healthit.chpl.dto.CertificationStatusEventDTO;
+import gov.healthit.chpl.dto.CertifiedProductDetailsDTO;
+import gov.healthit.chpl.dto.ProductDTO;
 import gov.healthit.chpl.entity.CertificationStatusType;
+import one.util.streamex.StreamEx;
 
 @Component()
 public class HistoricalStatisticsCreator {
     private static final Logger LOGGER = LogManager.getLogger("summaryStatisticsCreatorJobLogger");
```

Comment:
I see Eclipse mark this as not used either. I'm fine keeping it around because I wouldn't want to later want to log something, and forget to use the `summaryStatisticsCreatorJobLogger` thing, but I was wondering if the `@Log4j2` annotation has a way of specifying the logger we should use? I know there's an optional `topic`, but I don't know if that would map to our `getLogger` name. Maybe worth check if `@Log4j2(topic=summaryStatisticsCreatorJobLogger)` would work?